.github/workflows/CI.yml

```diff
@@ -83,5 +83,6 @@ jobs:
       - name: Codecov
         uses: codecov/codecov-action@v4
         with:
+          token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage.xml
-          fail_ci_if_error: false
+          fail_ci_if_error: true
```

Comment:
IMHO this should be the default, but Sentry disagrees 🤷

Reply:
Agreed